sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReduceFnRunner.java

```diff
@@ -108,7 +109,7 @@
   /**
    * Track which windows are still active and which 'state address' windows contain state
    * for a merged window.
-   *
+   * <p>
```

Comment:
This was intellij inserting <p> on empty comment lines. I switched that 'feature' off.